deploy/scripts/src/txma/test.ts

```diff
@@ -59,7 +59,7 @@ const profiles: ProfileList = {
       preAllocatedVUs: 1,
       maxVUs: 1500,
       stages: [
-        { target: 30, duration: '10m' } // Ramp up to 30 iterations per second in 10 minutes
+        { target: 500, duration: '10m' } // Ramp up to 30 iterations per second in 10 minutes
```

Comment:
```suggestion
        { target: 500, duration: '10m' } // Ramp up to 500 iterations per second in 10 minutes
```
Update comments to match values

Reply:
Done